windowsforms/datagridview/CSWinFormDataGridView/ResourceDeserializationExtensions.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System;
+using System.Drawing;
+using System.IO;
+using System.Resources;
+using System.Runtime.Serialization.Formatters.Binary;
+
+namespace CSWinFormDataGridView
+{
+    public static class ResourceDeserializationExtensions
+    {
+        static BinaryFormatter Formatter = new BinaryFormatter();
+
+        /// <summary>
+        /// Returns an image deserialized from a string resource to work around https://github.com/dotnet/corefx/issues/26745
+        /// </summary>
```

Comment:
Now that there's a fix for dotnet/corefx#26745 in the works, I can probably remove this.